content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx

```diff
@@ -101,7 +135,17 @@ if (user) {
 
 ## Filtering scalar lists
 
-In [2.15.0](https://github.com/prisma/prisma/releases/2.15.0) and later (Postgres only), you can [filter scalar lists](/reference/api-reference/prisma-client-reference#scalar-list-filters) <span class="api"></span> - for example, a list of `String` or `Enum` types. The following example returns all posts where the tags list includes `databases` _and_ `typescript`:
+<Admonition type="warning">
+
+Available for:
+
+- PostgreSQL in versions [2.15.0](https://github.com/prisma/prisma/releases/tag/2.15.0) and later
+- CockroachDB in versions [3.9.0](https://github.com/prisma/prisma/releases/tag/3.9.0) and later
+- MongoDB in versions [3.11.0](https://github.com/prisma/prisma/releases/tag/3.11.0) and later
+
+</Admonition>
+
+Use [scalar list filters](/reference/api-reference/prisma-client-reference#scalar-list-filters) <span class="api"></span> to search within scalar lists. The following example returns all posts where the tags list includes `databases` _and_ `typescript`:
```

Comment:
Nitpick: "search within scalar lists" is misleading. You're filtering model _records_ based on scalar list filters like "is this list empty",  "does this list have value X"... not sure `search` is the best generalization here, but I may be wrong.

Reply:
Hopefully updated version is better.